jekyll/_ccie/Artifactory.md

```diff
@@ -0,0 +1,61 @@
+# Artifactory 
+
+## Deploy
+
+Artifactory has great documentation explaining how to leverage their [REST API](https://www.jfrog.com/confluence/display/RTF/Artifactory+REST+API).
+
+We figure we use this space to highlight some sample projects that will show you how to best leverage using CircleCI and Artifactory together. 
```

Comment:
Suggested:

"We'll use this space to highlight some sample projects showing how to best leverage CircleCI and Artifactory together."